ouroboros-consensus-test/test-consensus/Test/Consensus/Node.hs

```diff
@@ -4,6 +4,22 @@
 {-# LANGUAGE OverloadedStrings   #-}
 {-# LANGUAGE RankNTypes          #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+-- | Tests for the DB marker and DB lock.
```

Comment:
If this is the case, i would find it reasonable to somehow rename the file to something like `DBLock.hs`

Reply:
Great point, but I could perhaps do it in a separate PR, to keep the scope of this one narrow. What do you think?